tests/conftest.py

```diff
@@ -0,0 +1,88 @@
+import os
+from pathlib import Path
+import pytest
+import json
+import requests
+
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk import Tracker
+from mailchimp3.mailchimpclient import MailChimpError
+
+from actions import config
+from actions.api.rasaxapi import RasaXAPI
+from actions.api.mailchimp import MailChimpAPI
+from actions.api.gdrive_service import GDriveService
+
+here = Path(__file__).parent.resolve()
+
+@pytest.fixture
+def tracker():
+    tracker = Tracker.from_dict(json.load(open(here / "./data/initial_tracker.json")))
```

Comment:
```suggestion
    with open("tests/data/initial_tracker.json") as json_file:
        tracker = Tracker.from_dict(json.load(json_file))
```

I think the relative path here works, you definitely want to use `with open()` anytime you read a file, because it will autoclose after you're done